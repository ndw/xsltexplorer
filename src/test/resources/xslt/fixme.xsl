<?xml version="1.0" encoding="utf-8"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                xmlns:xs="http://www.w3.org/2001/XMLSchema"
                exclude-result-prefixes="xs"
                version="2.0">

<xsl:output method="xml" encoding="utf-8" indent="no"
            omit-xml-declaration="yes"/>

<!-- FIXME: this is a test of a fixme comment, don't really fix me! -->

<xsl:template match="element()">
  <xsl:copy>
    <xsl:apply-templates select="@*,node()"/>
    <!--FIXME: Don't fix this either.-->
  </xsl:copy>
</xsl:template>

<xsl:template match="attribute()|text()|comment()|processing-instruction()">
  <xsl:copy/>
</xsl:template>

</xsl:stylesheet>
