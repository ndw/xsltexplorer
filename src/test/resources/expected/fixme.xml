<summary xmlns="http://nwalsh.com/ns/xslt/analysis" xml:base="..." imports="0" includes="0" variables="0" params="0" templates="2" functions="0" fixmes="2"><stylesheet id="S-22220" xml:base="..." imports="0" includes="0" variables="0" params="0" templates="2" functions="0" fixmes="2"><variables></variables><params></params><templates></templates><comments></comments><functions></functions><template id="S-22220-t-1" class="template" recursive="false" elsewhere="false" match="element()"></template><template id="S-22220-t-2" class="template" recursive="false" elsewhere="false" match="attribute()|comment()|processing-instruction()|text()"></template><comment id="S-22220-c-1" class="comment">FIXME: this is a test of a fixme comment, don't really fix me!</comment><comment id="S-22220-c-2" class="comment">FIXME: Don't fix this either.</comment></stylesheet></summary>